src/collective/elasticsearch/tests/test_catalog.py

```diff
@@ -1,13 +1,21 @@
 # -*- coding: utf-8 -*-
+import unittest2 as unittest
+
+try:
+    from Products.CMFCore.indexing import processQueue
+except ImportError:
+    def processQueue():
+        pass
 from collective.elasticsearch.hook import getHook
-from collective.elasticsearch.testing import createObject
 from collective.elasticsearch.testing import HAS_ATCONTENTTYPES
+from collective.elasticsearch.testing import createObject
 from collective.elasticsearch.tests import BaseFunctionalTest
 from collective.elasticsearch.utils import getUID
 from plone import api
-
-import unittest2 as unittest
-
+from plone.app.contentrules.actions.move import MoveAction
```

Comment:
Add `plone.app.contentrules` as a dependency on the test and test-archetypes extras in setup.py.

Reply:
it still needs to be done.